docs/user_api/index.rst

```diff
@@ -132,9 +139,16 @@ Remapping
 ---------
 .. autosummary::
    :toctree: generated/
+   :template: autosummary/accessor.rst
+
+   UxDataArray.remap
+
+.. autosummary::
+   :toctree: generated/
+   :template: autosummary/accessor_method.rst
 
-   UxDataArray.nearest_neighbor_remap
-   UxDataArray.inverse_distance_weighted_remap
+   UxDataArray.remap.nearest_neighbor
+   UxDataArray.remap.inverse_distance_weighted
    UxDataArray.nodal_average
```

Comment:
Can we remove this from remapping? This function is deprecated anyways.

Reply:
Yup!